src/app/static/src/app/components/outputTable/TableData.vue

```diff
@@ -55,9 +68,31 @@ export default defineComponent({
             }
             return filteredData
         });
+
+        const getFullData = (filteredData: any[]) => {
```

Comment:
Suggested rename - 'full data' sounds like it's going to be unfiltered to me. 
```suggestion
        const getDownloadData = (filteredData: any[]) => {
```